src/components/DeckBuilderDryRunView/index.js

```diff
@@ -10,10 +12,14 @@ export default props => {
   // purposes. The mode is restored to `AUTOMATIC` as soon as the 4th card has
   // been picked.
   const [mode, setMode] = React.useState(params.get('mode') || 'AUTOMATIC')
+  const [modifier, setModifier] = React.useState('0')
```

Comment:
I think we should name the modifiers rather than indexing them: e.g. `STRUCTURE_MANA`, `RAVEN_STRENGTH`, `CONSTRUCT_SPEED`.